packages/web-app-files/src/components/SideBar/Versions/FileVersions.vue

```diff
@@ -4,19 +4,20 @@
     <oc-table-simple v-if="!loading && hasVersion">
       <oc-tbody>
         <oc-tr v-for="(item, index) in versions" :key="index" class="file-row">
-          <oc-td width="shrink">
+          <oc-td width="shrink" class="file-icon">
```

Comment:
Why add this class here? Please work with test-ids if that's what you're trying to achieve here ;)

Reply:
now used `data-test-id` in place of table selectors.

Reply:
Please take a look at https://github.com/owncloud/web/discussions/5367#discussioncomment-955454, we've opted to go with `data-testid` in our repositories. Sorry for the duplicate effort on your side

Reply:
fixed 🚀